.github/workflows/dart.yml

```diff
@@ -20,5 +20,17 @@ jobs:
         run: pub get
       - name: Analyzer
         run: dartanalyzer lib test --fatal-infos --fatal-warnings
-      - name: Tests
-        run: pub run test
+      - name: Format
+        run: dartfmt --dry-run --set-exit-if-changed lib
+      - name: Run tests
+        run: pub run test_coverage --no-badge --print-test-output --min-coverage 0
+      #- name: Upload coverage to Codecov
```

Comment:
Do we need the commented-out part?

Reply:
Yeap, we need.

It's free service for badge and graphical representation of coverage.
https://codecov.io